README.md

```diff
@@ -158,8 +162,28 @@ As such it works with any executable that supports the following simple protocol
 
 Repeat forever:
 - read one line from standard input (file descriptor 0)
-- parse the line as a json object
-- execute the action, logging in standard output and in standard error (file descriptor 1 and 2)
+- parse the line as a json object, that will be in format:
+
+```{
```

Comment:
I think the contract states that `value` will be the one present, and any other fields in the JSON will be converted to environment variables.

Reply:
Ok, will fix it.